projects/step-core/src/lib/modules/dynamic-forms/components/dynamic-textfield/dynamic-textfield.component.html

```diff
@@ -1,28 +1,29 @@
-<step-expression-input
-  *ngIf="dynamic; else constantValueTemplate"
-  [label]="label"
-  [tooltip]="tooltip"
-  [placeholder]="placeholder"
-  [isParentInvalid]="!!_ngControl.invalid"
-  [isParentTouched]="!!_ngControl.touched"
-  [isParentDisabled]="isDisabled"
-  [showRequiredMarker]="showRequiredMarker"
-  [ngModel]="expression"
-  (ngModelChange)="onExpressionChange($event)"
-  [disabled]="isDisabled"
-  (toggleConstantValue)="toggleDynamicExpression(false)"
-  (blur)="onBlur()"
-></step-expression-input>
-
-<ng-template #constantValueTemplate>
+@if (dynamic) {
```

Comment:
I like the new @if directive! Looks cleaner than then the old else-template solution